Readme.md

```diff
@@ -1,32 +1,44 @@
+# trash
 
 [![Crates.io](https://img.shields.io/crates/v/trash.svg)](https://crates.io/crates/trash)
 [![Docs.rs](https://docs.rs/trash/badge.svg)](https://docs.rs/trash)
 
-## About
+Trash is a Rust library for moving files and folders to the OS's Trash or Recycle Bin.
 
-Trash is a Rust library that provides functionality to move files and folders to the operating system's Recycle Bin (or Trash or Rubbish Bin or what have you).
+Supported platforms:
 
-The library supports Windows, Mac and Linux.
+- Linux/*BSD: [The FreeDesktop.org Trash specification](https://specifications.freedesktop.org/trash-spec/trashspec-1.0.html)
+- macOS
+- Windows
```

Comment:
I like that you add *BSD however this library in its current state doesn't implement the fd.o trash specification itself. Instead it uses utility programs found on the system so this is misleading. I also prefer listing them in a single paragraph opposed to a markdown list format.